src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -465,6 +464,31 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
         resultsPage.hoverClickAndViewStudentPhotoOnHeading("1-2", Const.SystemParams.DEFAULT_PROFILE_PICTURE_PATH);
     }
 
+    @Test
+    public void testViewNoSpecificSection() {
+        ______TS("No Specific Section shown for recipient > question > giver with General Feedback");
+        resultsPage = loginToInstructorFeedbackResultsPageWithViewType("CFResultsUiT.instr", "Open Session", true,
+                "recipient-question-giver");
+        assertTrue(resultsPage.isSectionPanelExist(Const.NO_SPECIFIC_SECTION));
+
+        ______TS("No Specific Section shown for recipient > giver > recipient with General Feedback");
+        resultsPage.displayByRecipientGiverQuestion();
+        assertTrue(resultsPage.isSectionPanelExist(Const.NO_SPECIFIC_SECTION));
+
+        ______TS("No Specific Section shown giver > question > recipient with Feedback from Instructor");
+        resultsPage.displayByGiverQuestionRecipient();
+        assertTrue(resultsPage.isSectionPanelExist(Const.NO_SPECIFIC_SECTION));
+
+        ______TS("No Specific Section not shown for recipient > question > giver with no General Feedback");
+        resultsPage = loginToInstructorFeedbackResultsPageWithViewType("CFResultsUiT.instr",
+                "Session with sanitized data", true, "recipient-question-giver");
+        assertFalse(resultsPage.isSectionPanelExist(Const.NO_SPECIFIC_SECTION));
+
+        ______TS("No Specific Section not shown giver > question > recipient with no General Feedback");
```

Comment:
Typo? General Feedback should not get involved with giver > question > recipient right?

Reply:
Waiting for your comment :)